src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/AuditUtility.cs

```diff
@@ -374,19 +375,19 @@ private int ParseAuditLevel()
 
```

Comment:
Should we return `PackageVulnerabilitySeverity.Low` in this case?

Reply:
Very weird that the compiler didn't warn